src/soundbase.h

```diff
@@ -138,7 +156,7 @@ class CSoundBase : public QThread
 
     QString strSystemDriverTechniqueName;
     int     iCtrlMIDIChannel;
-    int     iMIDIOffsetFader;
+    std::array<CMidiCtlEntry, 128> aMidiCtls;
```

Comment:
Generally only Qt container classes (https://doc.qt.io/qt-5/containers.html) are used in the code, so it's probably stylistically better to stick with that.

Reply:
Working on it.